.github/workflows/build-binaries.yml

```diff
@@ -102,3 +107,24 @@ jobs:
         with:
           name: llvm-bins-${{ runner.os }}-${{ runner.arch }}
           path: ./${{ runner.os }}-${{ runner.arch }}-target-final.tar.gz
+
+      - name: Running Lit tests (Windows)
+        if: inputs.run_regression_tests && runner.os == 'Windows'
+        shell: 'msys2 {0}'
+        run: |
+          # Specify explicit targets on Windows to avoid Windows issues on MSYS2 platform
+          ninja -C './target-llvm/build-final' -v \
+            check-all verify-llvm-codegen-opt
+
+      - name: Running Lit tests
```

Comment:
To be fair to the preceding test, I think this should be named something like “Running Lit tests (Non-Windows),” or, depending on one’s feelings about Windows, “(Normal)” or “(Non-broken).